src/components/MetamaskLogin/style.sass

```diff
@@ -0,0 +1,49 @@
+@import "../common-styles/utils"
+
+.icon
+    position: relative
+    display: block
+    margin: 0 auto 1.5rem
+    width: 72px
+    height: 72px
+    border-radius: 50%
+    @include property(border, 1px, solid, 'l4')
+    @include var(background-color, 'l1')
+
+    :global(.circle-wrapper)
+        position: absolute
+        left: 0
+        top: 0
+
+    &::before
+        position: absolute
+        content: ''
+        width: 40px
+        height: 40px
+        left: 14px
+        top: 14px
+        background: no-repeat center
+        background-size: contain
+        background-image: url('https://cdn.iconscout.com/icon/free/png-512/metamask-2728406-2261817.png')
```

Comment:
@georgeciubotaru you should not do external images.
Please move it inside the component. Either as base64 in css or as js export/import.